### DISCUSSION TITLE
Addition of the "shutdown" state

### DISCUSSION THREAD

**Discussion by cloggedDrain** [teslamate-org/teslamate#801] [Ideas]
**What would you like to be added**:
During some service requests (as well as other situations), the car may be put in a "shutdown" state which teslamate does not seem to support 

**Why is this needed**:
The following error is created:
19:22:56.971 car_id=1 [warn] Error / unknown vehicle state "shutdown"

**Additional context**:
This is a rare situation, but my car is currently in service for a MCU2/HW3 upgrade and that is an error I'm receiving in my container

**Comment by Dulanic** [comment]
Good catch, this may be the variable used when the app shows as in Service.

**Comment by cloggedDrain** [comment]
I completely suspect that to be the case.
